service/prometheus/prometheus.go

```diff
@@ -6,6 +6,10 @@ import (
 
 const (
 	ClusterAnnotation = "giantswarm.io/prometheus-cluster"
+
+	// ClusterLabel is the Prometheus label used to identify jobs
+	// managed by the prometheus-config-controller.
+	ClusterLabel = "prometheus_config_controller"
```

Comment:
We need to make sure we don't completely bork the prometheus configuration - we have other stuff (kubernetes apiservers, host etcd, etc.) configured in there. We use this label to note which services the `prometheus-config-controller` manages.